### ISSUE TITLE
Add label property to search button in 'external' input 

### ISSUE THREAD

**Issue by SicParv1sMagna** [measuredco/puck#535]
It would be cool to add something like 'label' property into 'external' input to change search button label. In some cases it would be useful, for example, in cases where developer needs to add translations using i18n?

**Comment by chrisvxd** [comment]
Good idea. Suggest `searchButtonLabel` could be added as a [parameter](https://puckeditor.com/docs/api-reference/fields/external#params).

**Comment by SicParv1sMagna** [comment]
@chrisvxd added pr which solves the issue
